torch/profiler/_memory_profiler.py

```diff
@@ -1202,3 +1202,40 @@ def export_memory_timeline_html(
             with open(path, "w") as f:
                 f.write(html)
         remove(tmpfile.name)
+
+
+    def export_memory_timeline_json(self, path: str, device_str: str) -> None:
+        import numpy as np
+        import json
+        mt = self._coalesce_timeline(device)
+        times, sizes = np.array(mt[0]), np.array(mt[1])
+        if sizes.ndim == 1:
+            stacked = np.cumsum(sizes) / 1024**3 
```

Comment:
might be good to save 1024**3 as a define instead of using it over and over

Reply:
ok,you are right